### PR TITLE
Rebrand Chuck E. Cheese's

### PR DESCRIPTION
**Pull request by Cj-Malone**
I was surprised to see "Chuck E. Cheese's", but then I actually looked at some images and the fascia does have the `'s`. So I agree with the name, but I think the brand is `Chuck E. Cheese`.

Anyway, I might be wrong so I'll leave it to an American to decide.

### CODE REVIEW

**Comment by AntiCompositeNumber** [comment]
Yes, the `'s` was dropped starting in 2019 when they rebranded to be more "modern". I don't know how far the roll out of the rebranding has progressed especially since they went bankrupt again in 2020. I think it does make sense to update `brand=` and `name=` like we have with Dunkin' Donuts and other slowly-rebranding franchises.

**Comment by UKChris-osm** [comment]
I think this change would be better suited for all the tags, such as `display name` and `name`, rather than just the `brand` tag, as it seems the brand as a whole no longer includes the "'_s_".

**Comment by Cj-Malone** [comment]
Thanks everyone, agreed.

How do we feel about 2 categories on these?

File: data/brands/amenity/restaurant.json [hunk 1]
```diff
@@ -1155,18 +1155,18 @@
       }
     },
     {
-      "displayName": "Chuck E. Cheese's",
+      "displayName": "Chuck E. Cheese",
       "id": "chuckecheeses-85c92c",
-      "locationSet": {"include": ["ca", "us"]},
+      "locationSet": {"include": ["ca", "pr", "us"]},
```
Comment (arch0345) [inline data/brands/amenity/restaurant.json:1160]: According to their Wikipedia article, they have some locations outside of the US/Canada (Puerto Rico is included in the us country-coder polygon so it isn't necessary to include it): https://en.wikipedia.org/wiki/Chuck_E._Cheese#International_markets
```suggestion
      "locationSet": {"include": ["ae", "bh", "ca", "ch", "co", "gt", "hn", "mx", "pa", "pe", "qa", "sa", "sr", "sv", "tt", "us"]},
```

**Pull request merged**